agri-store-react/src/Components/Context/CartPageContext.js

```diff
@@ -10,10 +10,10 @@ function CartPageContext({ children }) {
   const { cart } = state;
 
   // auth token
-  localStorage.setItem(
-    "token",
-    `eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiJlMzIzZmY2MC1hMTUzLTQ0MTYtYmEyNS0zNDQ0ZGI1NjliOWMiLCJlbWFpbCI6ImFkYXJzaGJhbGlrYUBnbWFpbC5jb20ifQ._-fah2UEuueLmRHHl5uV4CYhiQdODX6neUkGbfTvtFM`
-  );
+  // localStorage.setItem(
+  //   "token",
+  //   `eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiJlMzIzZmY2MC1hMTUzLTQ0MTYtYmEyNS0zNDQ0ZGI1NjliOWMiLCJlbWFpbCI6ImFkYXJzaGJhbGlrYUBnbWFpbC5jb20ifQ._-fah2UEuueLmRHHl5uV4CYhiQdODX6neUkGbfTvtFM`
+  // );
```

Comment:
Remove this comment as there is no need of storing token explicitly if you've implemented Authentication